python/pyarrow/array.pxi

```diff
@@ -836,12 +836,6 @@ cdef class Array(_PandasConvertible):
             result = GetResultValue(self.ap.View(type.sp_type))
         return pyarrow_wrap_array(result)
 
-    def sum(self):
```

Comment:
Remove this `sum`?

Reply:
Actually removing this method is a backwards incompatible change. If we want to do that, I think we should deprecate it first (although it is certainly strange to only have a single reduction method exposed here)

For now I would maybe just pass through ``**kwargs`` (or leave as as)

Reply:
Added kwargs to array.sum(). Tested array.sum(min_count=0).